ingest-app/docs/api.md

```diff
@@ -694,9 +694,12 @@ curl -i -X DELETE \
 
 ### <a name="create-update-subscription"></a> Create / Update a Subscription
 
-Subscription concept can be created or updated by sending an HTTP PUT or POST with the metadata sent as data to the URL `%CMR-ENDPOINT%/providers/<provider-id>/subscriptions/<native-id>`. The response will include the [concept id](#concept-id) and the [revision id](#revision-id).
+Subscription concept can be created or updated by sending an HTTP POST or PUT with the metadata sent as data to the URL `%CMR-ENDPOINT%/providers/<provider-id>/subscriptions/<native-id>`. The response will include the [concept id](#concept-id) and the [revision id](#revision-id).
 
-If `native-id` is not specified in the request it will be generated. This is only supported for POST requests.
+If a native-id is not provided it will be generated. This is only supported for POST requests.
+POST requests may only be used for creating subscriptions.
```

Comment:
I am not sure on this. Should we allow POST to update? May want to ask other people's opinion.